lib/compress.js

```diff
@@ -3437,7 +3437,9 @@ merge(Compressor.prototype, {
                 }
                 var elements = [];
                 var consts = [];
-                exp.expression.elements.forEach(function(el) {
+                for (var i = 0, N = exp.expression.elements.length; i < N; i++) {
```

Comment:
Would you mind `len` instead of `N`? :ghost:

Reply:
sure